Mathlib/FieldTheory/RatFunc/AsPolynomial.lean

```diff
@@ -214,24 +214,33 @@ end HeightOneSpectrum
 
 section TrivialOnConstants
 
-variable {Γ : Type*} [LinearOrderedCommMonoidWithZero Γ] {v : Valuation (RatFunc K) Γ}
+variable {Γ : Type*}
 
 open Valuation
 
 /-- If a valuation `v` is trivial on constants then for every `n : ℕ` the valuation of
-`(monomial n a)` is equal to `(v (RatFunc.X)) ^ n`. -/
-lemma valuation_monomial_eq_valuation_X_pow
-    (hv : ∀ a : K, a ≠ 0 → v (algebraMap K (RatFunc K) a) = 1) (n : ℕ) {a : K} (ha : a ≠ 0) :
-    v (monomial n a) = v RatFunc.X ^ n := by
+`(monomial n a)` is equal to `(v RatFunc.X) ^ n`. -/
+lemma valuation_monomial_eq_valuation_X_pow [LinearOrderedCommMonoidWithZero Γ]
+    {v : Valuation (RatFunc K) Γ} (hv : ∀ a : K, a ≠ 0 → v (algebraMap K (RatFunc K) a) = 1)
```

Comment:
Here and everywhere else, isn't it better to replace `algebraMap K _ a` by `RatFunc.C a` (or even just `C a` if `RatFunc` is open)?